apps/dialtone-documentation/docs/.vuepress/baseComponents/DialtoneUsage.vue

```diff
@@ -12,7 +14,9 @@
     <div class="dialtone-usage__item dialtone-usage__item--dont">
       <h3 class="dialtone-usage__hd dialtone-usage__hd--dont">
         <dt-icon name="close" size="200" />
-        Don’t
+        <div class="dialtone-usage__label">
+          Don’t
+        </div>
```

Comment:
Not part of the Badge update, but applying the same approach here while I'm at it..